src/lib.rs

```diff
@@ -298,7 +274,7 @@ mod tests {
 
     #[test]
     fn generate_proof_test(){
-        let trie = verkle_trie_new();
+        let trie = verkle_trie_new(0,0);
```

Comment:
add test cases for all the variants

Reply:
Which among these should I consider?

1. Should I add the unit tests in their respective files?
2. Should I have unit tests in the `lib.rs` covering all the combinations
3. Both

Reply:
adding in the individual files would be better I guess?

Reply:
I've added unit tests for each file but I'm also thinking of adding full coverage test in the `lib.rs` too. Maybe after that I can remove tests from individual files if they seem to do the same tests